app/views/help.scala.html

```diff
@@ -371,29 +362,34 @@ <h3 class="govuk-heading-s" id="file-checks">Step 4: File checks</h3>
             </ul>
 
             <h3 class="govuk-heading-s" id="closure-metadata">Closure metadata</h3>
-            <p class="govuk-body">If you wish to transfer closed files and folders, you must add closure metadata.</p>
+            <p class="govuk-body">If you wish to transfer closed records, you must add closure metadata.</p>
             <p class="govuk-body">
-                Once you select the file or files to add closure metadata to, you must first confirm the closure status has been approved by the Advisory Council.
-                If it has been approved, you may proceed to add the following information:
+                Once you select the record to add closure metadata to, you must first confirm the closure status has been agreed with the Advisory Council and/or the National Archives.
+                If it has been agreed, you may proceed to add information to the following fields:
             </p>
             <ul class="govuk-list govuk-list--bullet">
-                <li>FOI decision asserted – this is the date of the Advisory Council approval</li>
-                <li>Closure start date – this is the date of the last meaningful change to the record</li>
-                <li>Closure period – this is the number of years the record is closed from the closure start date</li>
-                <li>FOI exemption code – you should search and select at least one exemption code per file</li>
-                <li>Title sensitivity – you should confirm if the title is sensitive to the public. If it is, you can add an alternative title</li>
-                <li>Description Sensitivity – you can only provide an alternative description if you have set a description on the Descriptive Metadata page first</li>
+                <li>FOI decision asserted – you should enter the date that the closure schedule was agreed with the Advisory Council and/or the National Archives. </li>
+                <li>Closure start date – you should enter the date of the last meaningful change to the record.</li>
+                <li>Closure period – you should enter the number of years the record must be closed from the closure start date.</li>
+                <li>FOI exemption code – you should search and select at least one exemption code per record</li>
+                <li>Is the title sensitive for the public? – under this question, you should confirm if the title is sensitive to the public. If it is, you can add an alternative title.</li>
+                <li>Is the description sensitive for the public? – under this question, you should confirm if the description is sensitive to the public. If it is, you can add an alternative description.
+                    You can only provide an alternative description if you have already added a description on the Descriptive Metadata page.</li>
```

Comment:
Should there be a line break between the sentences?

Reply:
It doesn't affect the way it's displayed on the page but I've removed the line break anyway